build.gradle

```diff
@@ -95,3 +110,15 @@ repositories {
 }
 
 apply from: 'extra_publication.gradle'
+
+// Pass to the external maven repositories after any modifications have been done during
+// the evaluation phase (as in an init.gradle script).  Needed for builds that are done
+// inside a corporate firewall.
+afterEvaluate {
+  String gradleMavenRepos = repositories.withType(MavenArtifactRepository).findAll {
+    it.name != 'MavenLocal' && it.name != 'Embedded Kotlin Repository'
```

Comment:
What does this has to do with Embedded Kotlin Repo?

Reply:
'MavenLocal' and 'Embedded Kotlin Repository' are both internal pseudo repositories that always exist even though they may be empty.  'Embedded Kotlin Repository' was added with the new Gradle Kotlin DSL in version 4.0.  In a non-Gradle Kotlin DSL build this Maven repository will exist but will not have any entries.   This was an easy way for me to get the list of all "real" external maven repositories ignoring the build in pseudo repos.  Neither one is an external Maven repository so should be excluded from being passed to the tests to use to pull down dependencies.